cdap-app-fabric/src/main/java/co/cask/cdap/app/store/Store.java

```diff
@@ -146,6 +207,20 @@ void setStartAndRun(ProgramId id, String pid, long startTime, long runTime,
   Map<ProgramRunId, RunRecordMeta> getRuns(Set<ProgramRunId> programRunIds);
 
   /**
+   * Fetches the active (i.e STARTING or RUNNING or SUSPENDED) run records against a given NamespaceId.
+   * @param namespaceId the namespace id to match against
+   * @return map of logged runs
+   */
+  Map<ProgramRunId, RunRecordMeta> getActiveRuns(NamespaceId namespaceId);
```

Comment:
seems like it would be better to update getRuns method to accept the set of status, rather than have new specific method for the active runs.. thoughts?